source/devices/J7_Family/linux/Release_Specific_Release_Notes.rst

```diff
@@ -105,10 +105,6 @@ U-Boot
 | Compiler Information: arm-oe-eabi-gcc (GCC) 13.3.0, aarch64-oe-linux-gcc (GCC) 13.3.0
 |
 
-.. note::
-
```

Comment:
Why is this note removed? I understand that currently there might not be any appended patches in meta-tisdk later, but in future if they do exists a note like this is still valid. It's a disclaimer note saying "may contain", looks harmless to me to keept it .

Reply:
@praneethbajjuri We have come up with a policy to not have any patches in meta-tisdk layer
Right @aniket-l :wink:

Reply:
But yeah, we can add the above disclaimer for meta-edgeai 

- https://git.ti.com/cgit/edgeai/meta-edgeai/tree/recipes-bsp/u-boot/u-boot-ti-staging?h=scarthgap
- https://git.ti.com/cgit/edgeai/meta-edgeai/tree/recipes-kernel/linux/linux-ti-staging?h=scarthgap

Reply:
Yeah, if PSDK is making an effort to no longer patch these components then there's no need to note this explicitly here. I do think it's worth noting in a Yocto specific section that one can check what patches are being applied to the kernel using bitbake to dump the `SRC_URI` variable for a given release.